src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
-body {
+* {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: #060B35;
 }
```

Comment:
Consider moving these styles into a separate file to keep your main file clean and clear